components/org/urbanscape/urbanscape-success.jsx

```diff
@@ -26,6 +30,18 @@ const UrbanscapeSuccess = () => {
         // eslint-disable-next-line
     }, []);
 
+    useEffect(() => {
+        if (verified === true && retrievedData !== null) {
+            setTimeout(() => {
```

Comment:
I don't really like `setTimeouts` in `useEffect` (especially when there is no `clearTimeout`). It introduces a hard-to-debug race condition where you change some values with some assumptions based on past values. If we introduce another `useEffect` that might set `isWaived` to `false`, this `setTimeout` might run after that and re-set it to true, even if it was not intended. I assume the `setTimeout` here was added to make sure the `retrievedData.credentials` has the credential, this can be achieved by doing some tweaks in the deps array to check for that credentials array:
```javascript
useEffect(() => {
  const credential = retrievedData?.credentials[0];
  if (verified === true && credential?.type?.includes('EquiNetCreditScore')) {
    setIsWaived(true);
  }
// eslint-disable-next-line
}, [verified, retrievedData?.credentials]);
```
This hook should trigger everytime there are changes to verified and the credentials array so it doesn't really care that on one trigger we don't have the credential because on the next trigger we should.